tests/e2e/specs/substance.js

```diff
@@ -61,6 +61,73 @@ describe("The substance page", () => {
       .find("text")
       .should("exist");
   });
+  it("should post not-loaded defined compounds", () => {
+    // Watch for patches
+    cy.route({
+      method: "POST",
+      url: "/definedCompounds",
+      status: 201,
+      response: {}
+    }).as("post");
+
+    // Select ill defined compound to access Ketcher
+    cy.get("#compound-type-dropdown").select("Defined Compound");
+
+    // Verify ketcher has loaded
+    cy.get("iframe[id=ketcher]")
+      .its("0.contentDocument.body")
+      .should("not.be.empty");
+    // Click Oxygen Button
+    cy.get("iframe[id=ketcher]")
+      .its("0.contentDocument.body")
+      .should("not.be.empty")
+      .then(cy.wrap)
+      .find("#atom")
+      .find("button")
+      .eq(3)
+      .click();
+    // Add CycloHexane to the canvas
+    //    cy.get("iframe[id=ketcher]")
+    //      .its("0.contentDocument.body")
+    //      .find("canvas#canvas")
+    //      .click();
+
+    cy.get("iframe[id=ketcher]")
+      .its("0.contentDocument.body")
+      .should("not.be.empty")
+      .then(cy.wrap)
+      .find("#canvas")
+      // create first node
+      .click();
+    // Save
+    cy.get("button:contains('Save Defined Compound')")
+      .should("not.be.disabled")
+      .click();
+
+    // Verify patch status and regex for structure
+    cy.get("@post").should("have.property", "status", 201);
+    cy.get("@post")
+      .its("request.body.data.attributes.molfileV2000")
+      // This regex accepts only a CycloHexane structure (metadata is excluded from cml tag)
```

Comment:
```suggestion
      // This regex accepts only an Oxygen structure
```
This comment makes more sense as you're only adding a lone H2O